introduction.md

```diff
@@ -0,0 +1,3 @@
+Hi, i'am kevin and i'am new to git and github, i would really
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.